### ISSUE TITLE
Autocommands.json seems to disable madmin

### ISSUE THREAD

**Issue by Axiangel** [Map-A-Droid/MAD#761]
I tried to add a autocommands.json file as described in the docs file by placing a slightly modified version of the example files in files/autocommands.json as described in the docs.
When i start MAD, Madmin is not running nor is MADdevice able to get status.
As soon as i remove the autocommands.json file and restart, madmin runs again.

Don't know if i'm doing anything wrong (most likely) or if there is a bug?
Is the location files/ even correct? Seems a bit odd to put it where all the .stat files are.

**Comment by sn0opy** [comment]
Does this still occur?

**Comment by Axiangel** [comment]
No, it seems to be fixed or caused by a misconfiguration on my part. I tried to reproduce in order to flush out any bugs, but failed - Everything is running smooth. 
Closing this for now.

**Comment by Axiangel** [comment]
It seems this is still a thing - atleast for me.

I create a job called 'autocommands.json' in the default MAD/files folder. When i restart MAD, the logs tell me that the autojob aws found and added. But the log never shows me a line, telling me that Madmin was started. And its not.
As soon as i remove the autojob file and restart, madmin works again.
This is the logline missing when i try to start with a autojob: 
```
 File "/home/axi/MAD/mapadroid/madmin/madmin.py", line 59, in madmin_start
    app.run(host=args.madmin_ip, port=int(args.madmin_port), threaded=True)`
```
The job itself it very basic and propely formatted:
```
[{
	"redo": true,
	"algotype": "daily",
	"algovalue": "21:00",
	"startwithinit": false,
	"origins": "ATV",
	"job": "ATV - Reboot",
	"redoonerror": false
}]
```
I got both -vvv and a -vvvvv log but they are not anonymized. I can send them over if needed. Please PM-

**Comment by Expl0dingBanana** [comment]
You can send the logs my way
